client/src/components/AdminContainer/style.css

```diff
@@ -0,0 +1,75 @@
+.ant-layout-header {
+  height: 15vh;
+  min-height: 119px;
+  background-color: white;
+  padding: 15px 130px 20px 50px;
+}
+
+.header-image {
+  display: flex;
+  text-align: left;
+}
+
+.header-image img {
+  width: 250px;
+  height: 50px;
+}
+
+.header-btn {
+display: flex;
+justify-content: flex-end;
+}
+.header-btn button {
+  width: 160px;
+  height: 35px;
+}
+
+.container {
```

Comment:
@MohammedAlghazali It might be good to have a more specific name, like `.admin-container`, to differentiate from non-admin containers that you might create for other pages in the future.

Reply:
yes it need modify